src/lib/image/bootstrap/busybox/busybox.c

```diff
@@ -0,0 +1,45 @@
+/* 
```

Comment:
I think I'm still waiting on this one (unless I missed it)... Should I do it, or do you want to?